zombienet/0001-basic-warp-sync/0001-test-warp-sync.toml

```diff
@@ -0,0 +1,29 @@
+[relaychain]
+default_image = "docker.io/parity/substrate:master"
+default_command = "/home/miszka/parity/05-zombienet-warp-sync/substrate-wrap.sh"
```

Comment:
Where is this script? Also, the path here should be valid in the image we use.

Reply:
fixed in: [0649849](https://github.com/paritytech/substrate/pull/12675/commits/0649849bd2088cfe774b1444b24a0ac4e388e9d2)